index.html

```diff
@@ -3016,11 +3016,42 @@ <h3 id="applying">
         repository</a> on GitHub.
       </p>
     </section>
-    <section id="internationalization">
+    <section id="internationalization" class="appendix">
       <h2>
-        internationalization
+        Internationalization
       </h2>
-      <div class="issue" data-number="323"></div>
+      <p>
+        It is expected that authors will localize the content of a manifest by
+        using one of the following options:
+      </p>
+      <dl>
+        <dt>
+          Dynamically setting the language:
+        </dt>
+        <dd>
+          This can include, for instance, asking the user what their preferred
+          language is and dynamically adding or replacing the manifest link
+          relationship to the document based on that language preference (e.g.,
+          using a URL like "manifest?lang=fr").
+        </dd>
+        <dt>
+          Using content-negotiation, or geo-ip, etc. on the server:
+        </dt>
+        <dd>
+          The server that hosts the web application could attempt to
+          predetermine the user's language by using a geo-ip look-up or by
```

Comment:
I could change to Geotargeting and link to https://en.wikipedia.org/wiki/Geotargeting